### PR TITLE
DOC: Add converters example to the loadtxt  docstring

### PR DESCRIPTION
**Pull request by WarrenWeckesser**

### CODE REVIEW

**Comment by WarrenWeckesser** [comment]
The Travis-CI tests have passed--click the *Details* link to check.  For some reason this page is not updating the status.

File: numpy/lib/npyio.py [hunk 1]
```diff
@@ -884,24 +884,35 @@ def loadtxt(fname, dtype=float, comments='#', delimiter=None,
     Examples
     --------
     >>> from io import StringIO   # StringIO behaves like a file object
-    >>> c = StringIO(u"0 1\\n2 3")
+    >>> c = StringIO(u"0 1\n2 3")
```
Comment (rossbar) [inline numpy/lib/npyio.py:887]:
```suggestion
    >>> c = StringIO("0 1\n2 3")
```

File: numpy/lib/npyio.py [hunk 2]
```diff
@@ -884,24 +884,35 @@ def loadtxt(fname, dtype=float, comments='#', delimiter=None,
...
     >>> np.loadtxt(c)
     array([[0., 1.],
            [2., 3.]])
 
-    >>> d = StringIO(u"M 21 72\\nF 35 58")
+    >>> d = StringIO(u"M 21 72\nF 35 58")
```
Comment (rossbar) [inline numpy/lib/npyio.py:892]:
```suggestion
    >>> d = StringIO("M 21 72\nF 35 58")
```

File: numpy/lib/npyio.py [hunk 3]
```diff
@@ -884,24 +884,35 @@ def loadtxt(fname, dtype=float, comments='#', delimiter=None,
...
     >>> np.loadtxt(d, dtype={'names': ('gender', 'age', 'weight'),
     ...                      'formats': ('S1', 'i4', 'f4')})
     array([(b'M', 21, 72.), (b'F', 35, 58.)],
           dtype=[('gender', 'S1'), ('age', '<i4'), ('weight', '<f4')])
 
-    >>> c = StringIO(u"1,0,2\\n3,0,4")
+    >>> c = StringIO(u"1,0,2\n3,0,4")
```
Comment (rossbar) [inline numpy/lib/npyio.py:898]:
```suggestion
    >>> c = StringIO("1,0,2\n3,0,4")
```

**Review by rossbar — COMMENTED** [review]
I think the added example is great - a nice, easy-to-understand illustration of the `converters` functionality.

Since you're touching up the `\\n`'s in the other examples - I don't think the prepended `u`'s are necessary any longer, unless setting the entire docstring as a raw string affects this somehow? Either way, feel free to ignore as the suggestions are tangential to the main point of the PR.

**Comment by WarrenWeckesser** [comment]
Good idea @rossbar, thanks.  I committed your suggestions, and the tests are running again.  Let's see if travis-ci is acting any better.

**Comment by seberg** [comment]
Thanks, good to have an example for converts.

**Pull request merged**